src/modules/sim-constants.js

```diff
@@ -2,213 +2,213 @@
 // Based in part on code from Apache Ripple, https://github.com/apache/incubator-ripple
```

Comment:
These constants comes from Ripple, and we use them in our built-in plugins simulation. But do we really want all of them? Some of them could be defined only as part of the plugin simulation, and not need to be defined here. There's no need to change anything related to this as part of this PR, I just bring the question :)